client/components/sites-dropdown/test/index.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import sinon from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import useFakeDom from 'test/helpers/use-fake-dom';
+import useMockery from 'test/helpers/use-mockery';
+import useFilesystemMocks from 'test/helpers/use-filesystem-mocks';
+
+describe( 'index', function() {
+	useFakeDom();
```

Comment:
I tried to remove it, but I got this error:

```
  components
    sites-dropdown
      index
        1) "before all" hook


  0 passing (2s)
  1 failing

  1) components sites-dropdown index "before all" hook:
     ReferenceError: window is not defined
      at Object.<anonymous> (node_modules/component-event/index.js:1:74)
      at Object.require.extensions.(anonymous function) [as .js] (node_modules/babel-register/lib/node.js:166:7)
      at Function.hookedLoader [as _load] (node_modules/mockery/mockery.js:111:12)
      at require (internal/module.js:20:19)
```

Currently my understanding is that `useFakeDom` creates the globals that we should expect to find in a real browser, using Automattic/react-test-env under the hood.

Honestly I didn't know the shallow vs deep rendering was a factor that should influence our choice on the usage of `useFakeDom`.